pandas/tests/arithmetic/conftest.py

```diff
@@ -82,7 +82,7 @@ def zero(request):
     --------
     >>> arr = pd.RangeIndex(5)
     >>> arr / zeros
-    Float64Index([nan, inf, inf, inf, inf], dtype='float64')
+    pd.Float64Index([nan, inf, inf, inf, inf], dtype='float64')
```

Comment:
This one needs to stay as it is because `doctest` will check that the output matches the output from `arr / zeros`, so for this file it's probably bets to import `Float64Index` at the top and then use that consistently (rather than using `pd.Float64Index`)

Reply:
I've reverted this now, and imported `Float64Index` directly (along with some other indexes used in the same place)